src/core/gui/dialog/toolbarCustomize/ToolbarAdapter.cpp

```diff
@@ -228,8 +228,9 @@ auto ToolbarAdapter::toolbarDragMotionCb(GtkToolbar* toolbar, GdkDragContext* co
     }
 
     if (d->type == TOOL_ITEM_ITEM) {
-        gtk_toolbar_set_drop_highlight_item(toolbar, d->item->createTmpItem(orientation == GTK_ORIENTATION_HORIZONTAL),
-                                            ipos);
+        GtkWidget *iconWidget = d->item->getNewToolIcon();
```

Comment:
nitpick: For consistency
```suggestion
        GtkWidget* iconWidget = d->item->getNewToolIcon();
```